pallets/messages/src/rpc/src/tests/rpc_mock.rs

```diff
@@ -0,0 +1,64 @@
+use sp_api::{ApiRef, ProvideRuntimeApi};
```

Comment:
This is what sets up the rpc client to support the `info` call on the runtime api